griptape/drivers/vector/redis_vector_store_driver.py

```diff
@@ -105,5 +110,29 @@ def query(
             )
         return query_results
 
+    def create_index(self, namespace: Optional[str] = None):
+        try:
+            self.client.ft(self.index).info()
+            logging.info("Index already exists!")
```

Comment:
Should it be an error?

Reply:
It shouldn't be an error since it's okay if the index exists right?

Reply:
Yeah, may be not an error then but a `warning`, since the method technically failed to create a new index.

Also, could you add ` -> None` to the signature, since we are not returning anything here?

Reply:
Done!